source/testing/testing-models.md

```diff
@@ -83,6 +83,23 @@ relationships, so you probably don't need to duplicate those tests.  You could
 look at the [Ember Data tests] for examples of deeper relationship testing if you
 feel the need to do it._
 
+### Proxy API Requests
+
+Ember CLI also allows you to forward requests to a proxy server. Set the following
+properties in your `package.json`:
+
+```package.json
+{
+  "APIMethod": "proxy",
+  "proxyURL": "http://apiserver.dev:3000",
+  "proxyPath": "/api"
+}
+```
+
+This proxies all requests for a URL starting with `proxyPath` to the proxy server.
```

Comment:
Also, doesn't this content apply to the development server as well? I'm not sure testing is the right place to introduce this - maybe this text should go in the Models section somewhere, and then be referenced here?